### PR TITLE
Integration of a Doodle-like functionality in OAE

### PR DESCRIPTION
**Pull request by Orodan**
Hi,

I would like to propose you the work we've done, Mathilde Guerin, Frederic Dooremont and myself. We integrated a Doodle-like functionality in OAE as a new type of content, to be able to schedule meetings in OAE.

We inly add a new file called lib/api.loodle.js in this module, which is used in the oae-loodle module (see https://github.com/oaeproject/Hilary/pull/1290) for the tests for example.

I hope you appreciate this work and feel free to contact us if you have any questions of remarks.

Best regards

Jimmy

---
This change is [Reviewable](https://reviewable.io/reviews/oaeproject/oae-rest/29)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,35 +1,96 @@
 {
```
Comment (sallakarppinen) [inline package.json]: This should not be included in the PR.

File: node_modules/underscore/LICENSE [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2009-2014 Jeremy Ashkenas, DocumentCloud and Investigative
```
Comment (sallakarppinen) [inline node_modules/underscore/LICENSE]: No dependencies should be included in a PR.

File: .npmignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules/
```
Comment (sallakarppinen) [inline .npmignore]: Would take this out in favour of .gitignore.

**Comment by brecke** [comment]
Aborted.
